UX-Guide-Metadata/draft/techniques/onix-metadata/index.html

```diff
@@ -759,8 +759,188 @@ <h4>Instructions</h4>
+						</dd>
+					</dl>
+					
+					<h5>Variables setup</h5>
+					<ol class="condition">
+						<li><b>LET</b> <var>onix</var> be the result of calling <a href="#pre-processing">pre processing</a> given <var>onix_record_as_text</var>.</li>
+						<li><b>LET</b> <var>text_to_speech_hinting</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>onix</var>, <code class="xpath">/ONIXMessage/Product/DescriptiveDetail/ProductFormFeature[ProductFormFeatureType = "09" and ProductFormFeatureValue = "21"]</code>.</li>
+						<li><b>LET</b> <var>color_not_sole_means_of_conveying_information</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>onix</var>, <code class="xpath">/ONIXMessage/Product/DescriptiveDetail/ProductFormFeature[ProductFormFeatureType = "09" and ProductFormFeatureValue = "25"]</code>.</li>
+						<li><b>LET</b> <var>high_contrast_between_text_and_background</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>onix</var>, <code class="xpath">/ONIXMessage/Product/DescriptiveDetail/ProductFormFeature[ProductFormFeatureType = "09" and ProductFormFeatureValue = "26"]</code>.</li>
+						<li><b>LET</b> <var>ultra_high_contrast_between_text_and_background</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>onix</var>, <code class="xpath">/ONIXMessage/Product/DescriptiveDetail/ProductFormFeature[ProductFormFeatureType = "09" and ProductFormFeatureValue = "37"]</code>.</li>
+						<li><b>LET</b> <var>visible_page_numbering</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>onix</var>, <code class="xpath">/ONIXMessage/Product/DescriptiveDetail[ProductFormDetail = "E205"]</code>.</li>
+						<li><b>LET</b> <var>high_contrast_between_foreground_and_background_audio</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>onix</var>, <code class="xpath">/ONIXMessage/Product/DescriptiveDetail/ProductFormFeature[ProductFormFeatureType = "09" and ProductFormFeatureValue = "27"]</code>.</li>
+						<li><b>LET</b> <var>without_background_sounds</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>onix</var>, <code class="xpath">/ONIXMessage/Product/DescriptiveDetail[ProductFormDetail = "A312"]</code>.</li>
+					</ol>
+					
+					<h5>Instructions</h5>
+					<ol class="condition">
+						<li>
+							<span><b>IF</b> <var>text_to_speech_hinting</var> <b>OR</b> <var>color_not_sole_means_of_conveying_information</var> <b>OR</b> <var>high_contrast_between_text_and_background</var> <b>OR</b> <var>ultra_high_contrast_between_text_and_background</var> <b>OR</b> <var>visible_page_numbering</var> <b>OR</b> <var>high_contrast_between_foreground_and_background_audio</var> <b>OR</b> <var>without_background_sounds</var>:</span>
```

Comment:
@gregoriopellegrino Instead of always having these complex "or'ed" If statements instead just always set the "clarity" variable as an empty array and then at the end have a single If (Not empty) array then do xyz.
I think that's a cleaner approach than checking each variable every time if it has been set or not.

If you agree we should adjust the other places in this document where you do something similar.

Reply:
Done, good suggestion